src/components/modals/AddNewEmployee/index.tsx

```diff
@@ -9,10 +9,33 @@ import Modal from '@elements/Modal';
 import TextInput from '@elements/Inputs/TextInput';
 import AddButton from '@elements/Buttons/AddButton';
 import DiscardButton from '@elements/Buttons/DiscardButton';
+import { useState } from 'react';
+import { useApi } from '@hooks/useApi';
+import { addNewEmployee } from '@api/employeeService';
 
 const AddNewEmployee = () => {
     const { t } = useTranslation();
     const dispatch = useDispatch();
+    const firstName = '';
+    const lastName = '';
+    const [payload, setPayload] = useState({
+        fullname: `${firstName}${lastName}`,
```

Comment:
I think you should have space between first and last name.
fullname: `${firstName} ${lastName}`